modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -161,6 +172,14 @@ const RichText = ({
 
 			setCurrentAvailableLocales(availableLocales);
 
+			if (
+				evaluable &&
+				portletNamespace === ddmFormAdminPortlet &&
+				skipsChangeValidation(fieldName)
+			) {
+				return;
+			}
+
```

Comment:
After studying the case, I could see that not evaluating a field was already part of some corrections:

[LPS-201682 Error Message is copied over between Form Fields](https://github.com/liferay-forms/liferay-portal/pull/2400/files)
[LPS-202852 Update field when storageTypes field changes](https://github.com/liferay-forms/liferay-portal/pull/2406/files)
[LPS-203011 Do not update page only when changing error message](https://github.com/liferay-forms/liferay-portal/pull/2411/commits/63556f84e7616b80ff876740e102bfc7e421b65f)
[LPD-1783 Rename needsPageUpdate to needsPageEvaluation because what it most importantly does is to skip EVENT_TYPES.FIELD.EVALUATE when it returns false](https://github.com/brianchandotcom/liferay-portal/pull/148471/commits/5194959cbd36170563ee093bfd16a66c4b4caf66)

This helps us understand the story behind this [conditional](https://github.com/liferay/liferay-portal/blob/fafb2e6c9e66d62336eeeaa2298703536263e4ca/modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js#L108) better.

But for me it's strange that we have an evaluable variable, and yet we need other functions and conditions to decide whether we are going to evaluate a field or not.

I would like to know together with our BE people if there is a way to return this evaluable property in a more accurate way. So that we wouldn't have to try to do this management on the FE side.

cc: @marcelabc @carolmariaabb

Reply:
sure, thanks @igor-franca! I will do that shortly.